xiaogpt/xiaogpt.py

```diff
@@ -229,12 +238,20 @@ def _change_prompt(self, new_prompt):
     async def get_latest_ask_from_xiaoai(self, session):
         retries = 2
         for _ in range(retries):
-            r = await session.get(
-                LATEST_ASK_API.format(
-                    hardware=self.config.hardware,
-                    timestamp=str(int(time.time() * 1000)),
+            try:
+                timeout = ClientTimeout(total = 15)
```

Comment:
it seems unused?

Reply:
Sorry. It's a typo when fixing formatting. Now timeout is added to "get()" as a parameter.

Reply:
Sorry. It's a typo when fixing formatting. Now timeout is added to "get()" as a parameter.